apis/apiextensions/v1/composition_types.go

```diff
@@ -295,6 +295,10 @@ type ContainerFunction struct {
 	// +kubebuilder:default="20s"
 	Timeout *metav1.Duration `json:"timeout,omitempty"`
 
+	// Secrets for pulling function images.
+	// +optional
+	ImagePullSecrets []ContainerFunctionImagePullSecret `json:"imagePullSecrets,omitempty"`
```

Comment:
```suggestion
	ImagePullSecrets []corev1.LocalObjectReference `json:"imagePullSecrets,omitempty"`
```

Any reason not to use the same type as [packagePullSecrets](https://github.com/crossplane/crossplane/blob/580d15ac8ac79e72acec7f95a70c8ddd9026d6b8/apis/pkg/v1/package_types.go#L43)?